<launch>
  <arg name="bag_name" default="V1_02_medium.bag"/>
  <node pkg="sdd_vio"
    type="sdd_vio_bag_reader"
    name="sdd_vio_node"
    output="screen">
    <param name="~left_image_topic" value="/cam0/image_raw"/>
    <param name="~right_image_topic" value="/cam1/image_raw"/>
    <param name="~imu_topic" value="/imu0"/>
    <rosparam command="load" file="$(find sdd_vio)/config/vo_param_euroc.yaml"/>
    <rosparam command="load" file="$(find sdd_vio)/config/camchain-imucam-euroc.yaml"/>
    <param name="bag_file" value="$(find sdd_vio)/bagfiles/$(arg bag_name)"/>
    <remap from="~cam" to="odom_vio"/>
  </node>
</launch>
